source/_integrations/yardian.markdown

```diff
@@ -26,3 +26,14 @@ There is currently support for the following platform within Home Assistant:
 During the configuration, you will have to manually set the **Host** and the **Access Token**. You can find them inside your [Yardian App](https://yardian.com/app/).
 
 ![Yardian Host/Token Location](/images/integrations/yardian/yardian_config_flow.jpg)
+
+## Services
+
+### yardian.start_irrigation
+
+Start a zone for a given number of minutes. This service accepts an Yardian Zone switch entity and allows a given duration.
+
+| Service Data Attribute | Optional | Description                                           |
+| ---------------------- | -------- | ----------------------------------------------------- |
+| `entity_id`            | no       | The Yardian Zone switch to turn on.                   |
```

Comment:
This is an entity service, meaning it accepts a target. This also means `entity_id` is actually optional (as one could alternatively target an area or a device).

../Frenck

Reply:
got it!